### ISSUE TITLE
Will VMLens work with TestNG or only junit?

### ISSUE THREAD

**Issue by JFK-DXML** [vmlens/vmlens#13]

**Comment by ThomasKrieger** [comment]
Yes vmlens will also work with testng. Note that I am currently refactoring large parts of it so it will probably make sense to wait till I am finished before you try it. Hope it wont take that long till I am finished

**Comment by JFK-DXML** [comment]
> Yes vmlens will also work with testng. Note that I am currently refactoring large parts of it so it will probably make sense to wait till I am finished before you try it. Hope it wont take that long till I am finished

Excellent news. Thanks.


I am using the following configuration in my Maven POM

<plugin>
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-surefire-plugin</artifactId>
<configuration>
	<includes>
		<include>none</include>
	</includes>
</configuration>
</plugin>
<plugin>
      <groupId>com.vmlens</groupId>
      <artifactId>interleave</artifactId>
      <version>1.1.5</version>
      <configuration>
          <doNotTrace>
              <method>org.testng.**</method>
            </doNotTrace>
      </configuration>
      <executions>
	      <execution>
		      <phase>test</phase>
		      <goals>
			      <goal>test</goal>
		      </goals>
	      </execution>
      </executions>
</plugin>


I noticed without the doNotTrace things went a lot slower and needed a lot more memory. Obvious I know but I thought others might find this useful.

**Comment by ThomasKrieger** [comment]
Yes it works now with version 1.2.1. See https://github.com/vmlens/vmlens/blob/master/test-vmlens-testng/src/test/java/com/vmlens/test/maven/plugin/TestVolatileField.java
